spec/observables/dom/ajax-spec.ts

```diff
@@ -763,12 +814,11 @@ describe('Observable.ajax', () => {
     const request = MockXMLHttpRequest.mostRecent;
 
     expect(() => {
-      request.onprogress((<any>'onprogress'));
+      request.upload.onprogress((<any>'onprogress'));
```

Comment:
On second thought. I think this can be remedied at a later time. This is fine for now.